src/pages/komodo-defi-framework/api/v20-dev/non_fungible_tokens/enable_nft/index.mdx

```diff
@@ -0,0 +1,98 @@
+export const title = "Komodo DeFi Framework Method: Enable NFT";
+export const description =
+  "The enable_nft method allows you to activate NFT-like tokens on the platform.";
+
+# enable\_nft
+
+The 'enable_nft' method activates NFT-like tokens on the platform. Before using this method, you first need to use the [enable\_eth\_with\_tokens](/komodo-defi-framework/api/v20/enable_eth_with_tokens/) method.
+
+| parameter                                  | Type    | Description                                                                                        |
+| ------------------------------------------ | ------- | -------------------------------------------------------------------------------------------------- |
+| ticker                                     | string  | Ticker of the NFT-like token coin.                                                               |
+| activation\_params | object | object encapsulating all parameters for activation. |
+| activation\_params.provider | object | Defines available NFT providers and their configuration. |
```

Comment:
I suppose you can leave one record `| activation\_params.provider |`, as there is one field in `activation\_params`. However, check how object types are described in repo. You should add nft provider structure in nft common structures folder and provide it in the field description

examples
![Screenshot 2024-07-10 at 16 29 06](https://github.com/KomodoPlatform/komodo-docs-mdx/assets/52405288/8c8f8145-d078-4c86-88f4-6577b017f26b)

![Screenshot 2024-07-10 at 16 29 31](https://github.com/KomodoPlatform/komodo-docs-mdx/assets/52405288/c3f0991a-ee01-4bf5-9dbf-2a393e242499)